packages/@webex/plugin-cc/src/services/core/WebSocket/WebSocketManager.ts

```diff
@@ -4,8 +4,9 @@ import {SubscribeResponse} from '../../config/types';
 import LoggerProxy from '../../../logger-proxy';
 import workerScript from './keepalive.worker';
 import {KEEPALIVE_WORKER_INTERVAL, CLOSE_SOCKET_TIMEOUT} from '../constants';
+import {EventBus} from '../EventBus';
 
-export class WebSocketManager extends EventTarget {
+export class WebSocketManager {
```

Comment:
Now that we use the EventBus, we no longer need to extend the EventTarget, we can just emit via the bus instance.